Xkcd-Comics/ViewModels/ComicsVM.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 
 class ComicsVM: ObservableObject {
-    @Published var comicObject = Comic(num: 1, title: "Default", img: "Default", alt: "Default", year: "Default", month: "Default")
+    @Published var comicObject: Comic = Comic(num: 1, title: "Default", img: "Default", alt: "Default", year: "Default", month: "Default")
```

Comment:
Also, a point about naming: You are using redundant descriptive suffixes of your properties. You don't need to specify that something is an "object", because everything is an object. Likewise, you don't need to specify that something is a "list", but instead that is typically discerned through the clear and consistent use of singular and plural. 

The convention is to use

* `comic = Comic(...)`
* `comics = [Comic]`